godot-core/src/storage.rs

```diff
@@ -101,6 +103,13 @@ mod single_threaded {
             })
         }
 
+        pub fn get_gd(&self) -> Gd<T>
+        where
+            T: Inherits<<T as GodotClass>::Base>,
```

Comment:
This bound is not really meaningful -- an object inherits its `Base` by definition.

Is it really needed (because of `cast()`)? If yes, then maybe document that 🙂

Reply:
it is needed unfortunately, we dont require a class to inherit from its base, even though we always do.